src/components/filter/row/MultipleStringFIlterRow.tsx

```diff
@@ -74,6 +75,15 @@ export default function MultipleStringFilterRow({ config }: MultipleStringFilter
 		input.current?.focus();
 	};
 
+	const focusBubble: React.KeyboardEventHandler<HTMLInputElement> = e => {
+		if (e.keyCode === KeyCodes.LEFT) {
```

Comment:
Before switching we should also check if input caret at the end or at the start of value.